wagtail/admin/templates/wagtailadmin/shared/formatted_field.html

```diff
@@ -0,0 +1,61 @@
+{% load wagtailadmin_tags i18n %}
+{% comment "text/markdown" %}
+
+    The field template is very flexible to cater for a wide range of use cases. It has two main attributes that support different use cases:
+
+    1. `field` to render a Django form field. This may be done through the `{% formattedfield my_field %}` template tag, or a direct `include` of this template.
+    2. `rendered_field` to render arbitrary HTML. This may be done through the tag pair `{% rawformattedfield %}<select>[…]</select>{% endrawformattedfield %}`,
+         or a direct `include` of this template.
+
+    If both `field` and `rendered_field` are passed, `rendered_field` will be used as the actual form rendering, but metadata
+    for the surrounding elements (such as help text and ID) will be picked up from `field`.
+
+    A unified template for both use cases is messy, but it guarantees we are keeping form field styles the same everywhere.
+{% endcomment %}
+
+<div class="w-field__wrapper {{ classname }}" data-field-wrapper>
+
+    {# Render custom label attributes if provided, or the bound field’s label attributes otherwise. #}
+    {% if show_label %}
+        <label class="w-field__label{% if sr_only_label %} w-sr-only{% endif %}"{% if label_for %} for="{{ label_for }}"{% endif %}{% if label_id %} id="{{ label_id }}"{% endif %}>
```

Comment:
```suggestion
        {# Add an id to the label so we can use it as a descriptor for the "Add comment" button. #}
        <label class="w-field__label{% if sr_only_label %} w-sr-only{% endif %}"{% if label_for %} for="{{ label_for }}"{% endif %}{% if label_id %} id="{{ label_id }}"{% endif %}>
```

This felt like an important comment to me, as it’s otherwise really unusual to have an id on a label element.